competitions/s1floods/benchmark_tutorial.ipynb

```diff
@@ -0,0 +1,2029 @@
8USXcGFP0ZTlRCub2HXjo7kGaSRBoWmLMhYaWOX70MC4eW+ClCkFBPtslW2iRiTJHlKFJCFKmY4fAXXfVf+Ty1yp8dkkQieqlEnon4A5j5HZrmgFKpc2ZV9tSq2blV30QrMsrWmMjoII3G5H6AjoMlPManTLs7o6ydPrMKN4DhmLRxg9hYxSwm51ZJSJU26V9qlYtRkUlTCysZUn5mVYvCDYm1M6v08YtWhamH8cyqgLXyPnqnhFvZ+qJV8wpZuQv2c6sSRUUIXJysRnXxAcG5Vd3kIw6sHhNnneX7n1K1+ki9M6tSSyEUGiTJU6se41sV5CKlXTwiWL9ELv+MLPuMD4Q216/8a+4/+m0ADoMrWM4Fq2Y7TGpXSA7LbBmzc6tOsQSXcZyi6l9uleWMSSXxBasSpXRm1eTLrcokBSFPCbQLVpku7lGTB+lvcLgQWX1bJbQvEcRHuFKdajmmXdnBy0VkTSb1UmLJoqYNyT5nVahWyCpb51ZVmOnX0Z9ZtcORr5Cvf4Xtk1ex9fuU3lJwzRvnVhUczLcQJ9tExxPsbvrcquMOWglyVeOgf4OwKmJJ71Kvj2D6OatyB7kjsGbcJilqvH3pLoMnNnPrxnOrBtKZVYMZTmR+ziqZNCowZkOiZ1YtaImf/RVWjX6uXv5SDJlcMWFlGb/TJMirRBUZfXaC2cuIt0egFcSFRZxJ5GGGWmRIWYpr3UTywLQjnLhHSXRQtAnp7DFqlJAoGZLks1V5wOniEiP7G9jFIUZ4SCwYTIcGbqVKybTQ+w/wHJNQ0VCTMUbsYaYLokVCzd5DbJWZjDVO0zXCapd6foCYiYRmk1i2adRdcsnArawgjh+jxwP8kwQ5P6ba0KnKIbXoGFuZINkGsSeiL+7SyG3ioUdgtklrDfJSE01y8I8yAjQqqcBCW2ZaLKO3DerWBP9kxmhqslTappP9GcNgk6m2glSVkeMEo6ziKj0SR8YWZwSKRXRlGXu+T57lCGFKmmtoaoZRg57RZ9p/nz2/y6EgU6lt8NXrH7HlfZ842EDqgCZ6vDv8DbYbr1FfDVh4KnVhH9lNEY8myGLMRFghVirYwgy/toazP0fSRIrJlFnRoiLGlMMDFCkiLtVQEh9h4hM7BdZ8h3z1KpKRU9IdgrxMvfwYT1/hQHkLYZZwclpQurGEpfdJ+qe4gwy3oZIUKofOV0ESuWI/IqrYVNNDchSEIqMqDFBNhbTdozy+T3CiU1QEtINdvL7MfHWF0opKe0nHPH5Mu3eITMDJyTIV55Rl6f/J8dK3iDwf6fEuy+M/p14aMPJWebT7Nn13gyjSuNH/iEv8Ba3OZcLSK3z48S28+Sba8DH6nYTX2x/Qlgb0Ap+7d1c4/uoyS+X36NonHDa7fKaV0fsfcmf2bZzDGPH+FDubsOisExUgFTKGN4NcQg8nWKoLlklp/imxVkJNI6TEB80gnw8YuivkhYi20qM13ac6ffiLrvqvfM6sWrpglYQ+O31ulV4Qc8Gq/Nyq0kvIboFpx8+tUifk80coF60q3+dkcYWx/XXKxSF6dEgiGEyHJl6lgmWU0Af38VyLUFZRkwlG4mKmzplV5T3EVoXJSOU0WyeudKjlBwjnViWyTb1+cMGqR+jx8JlVlc9ZJZYNYkdAn3/yRavsp1al51aJz61qPbVqemaVvU0n+/Nzq1aRqgpylGBUVFz5glXqBavSDCE6t0rLMGsCS/ops/773OdsyDw8XOftax+z6X2fJNhA6oImurw7+vUzq1YCFr5KXThAdhPEoxGymL5oVXUN9+DLrDpEliKiUg058i5YtUu2egXtmVX2BavehlnK6UmBfWMZSzsl6Z/gDnLchkaSf7lV2QtWqWTtDuXxPcJzq/T9XZyBzGx1GXtVpdXVsI4f0e7un1l1vExlccKy+J9/zqo/o26PGHsrPNx9m8G5VTcHH7JV/JBO5zK+9QoffXwLbx6gDbfRP0l4vXXBqk9XOH57mSX7XbqlUw6bvQtWfQvn6Nyq/NwqnltV5BLGuVXCC1aFSElAoRnIiwHDxQoFIuryU6se/Fy9/KUYMpIms959TH1lxtC5Siv/GMvYxU8aCKUm84nJYlKhZTnoWggbPfJqmdBVKQ1uY3j7uOkGXsOk7KdUDj+hZRwy33yVRe9tFO2APKsQRyJFMKau3yfIVCqTOzj211nYL6GUR0jTCXa6y+SJRBYaZDWTPsvIsoosVXD1gq5wnyI+RNIEpvYK1ZbPfNfH8B4j5QaDWMFOR2hbBhNphem8wpX2EywhwTuuE/QuIzZXcYoutfge1tEnpJKF2CojpT7uIsXK92mMHxMWJXQtxK9cot7NkCcPsBWHxFhiJ/oNUrVNCZ+a95BMhjwoUPwpg9IGQlaQaDpir8qwaCCIOYUYMPKqCEGIef82QsuC7u8ivlVi4/6Y7cMWn+Ut3O0xb1/6GV9X/iUf7Hwda03A0mr0t95hNFhitfYIL19iP+pQaBly5CEHGRVxD1nTUdWY+H6I0K3ilzo0zD7TtMdoHlPsjGmuDPCWN5iXNsjjEdpSirG/i1KRKUljMtVElhWkOCecFPhCiBuapJUGq+1T4u0jDj4qSEOLQNJZffQuCTaqXiDVUyRFoFhEzAKdQhBx8yXK97bpTO9Dq8S8qOJIXZTFDM3MqLoPmI6uQT1gadNFD4458bYIvGvIR1NuKH9MNB1xEn+Vq/M/5JXWbfSVlLtpje3TLSzvAEGok88WzBbw8LM2an2Kv3uIF5aIpxohCnvRForoUBN95MSl399kY2UfVS+wpxPsaszIW2deew139zHldQ8/MpEsFS8roxPQCEb4lS7i0CUsoF56Qik5ZixvMa7cYB6WsYfbSO6CMHBRWzrJoxOSeYAvln7RVf+Vz5lV29RX5gydK2dW6bt4aQPBajKfmSzGF6za7JJXK4SOjDW4jeEdfM6qOzSNI2abr+H0rqGoBxRZhSQSyYMRdf0eQapRmd7GKX8Dx76JEo6QZhPseI
```

Comment:
Can you post the output of this cell running on the Planetary Computer?

Previously, `pip installing` some additional package that depended on torch would wipe out our existing (conda-based. I've since upgraded the package versions, but haven't tried that again.

If this causes issues, we might `!mamba install -y -c conda-forge albumentations` instead.

Reply:
I haven't encountered any issues installing `albumentations`. I'll post the output of these pip installations so you can take a look to ensure we aren't wiping anything unexpected.